### PR TITLE
[DOC-474] [2024.1.3] Added another parameter for the flag `ysql_hba_conf_csv`

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /stable/yugabyte-platform/security/authentication/oidc-authentication-aad/

### CODE REVIEW

**Review by Vars-07 — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-aad.md [hunk 1]
```diff
@@ -199,6 +199,7 @@ The `ysql_hba_conf_csv` flag must be set to support using JWTs for authenticatio
 - `jwt_audiences` - the audience or target app for the token, which in this case is the client ID of the application you registered.
 - `jwt_matching_claim_key` - the email attribute you set (for example, `preferred_username`). Optional if you aren't using the default Subject claim values.
 - `jwt_jwks_path` - The JSON Web Key Set (JWKS) is a set of keys containing the public keys used to verify any JWT. These can be uploaded as entries in a single file. When configuring the flag in YugabyteDB Anywhere, click **Add JSON web key set (JWKS)** to upload the JWKS.
+- `jwt_jwks_url` - Specifies the URL which YugabyteDB can retrieve the JWKS for verifying JWTs. This parameter is an alternative to `jwt_jwks_path`, where you need to provide the actual key content. Either `jwt_jwks_path` or `jwt_jwks_url` must be set to enable JWT verification in YugabyteDB.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-aad.md:202]:
```suggestion
- `jwt_jwks_url` - The URL where YugabyteDB can retrieve the JWKS for verifying JWTs. This parameter is an alternative to `jwt_jwks_path`. You must set either `jwt_jwks_path` or `jwt_jwks_url` to enable JWT verification in YugabyteDB.
```

Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-aad.md:202]: These bullets might be attractive as a table

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-aad.md [hunk 2]
```diff
@@ -192,13 +192,16 @@ For information on configuring flags in YugabyteDB Anywhere, refer to [Edit conf
...
-- `jwt_issuers` - the first part of the discovery URL (`login.microsoftonline.com/<tenant_id>/v2.0`)
-- `jwt_audiences` - the audience or target app for the token, which in this case is the client ID of the application you registered.
-- `jwt_matching_claim_key` - the email attribute you set (for example, `preferred_username`). Optional if you aren't using the default Subject claim values.
-- `jwt_jwks_path` - The JSON Web Key Set (JWKS) is a set of keys containing the public keys used to verify any JWT. These can be uploaded as entries in a single file. When configuring the flag in YugabyteDB Anywhere, click **Add JSON web key set (JWKS)** to upload the JWKS.
+The `ysql_hba_conf_csv` flag must be set to support using JWTs for authentication. The parameters to include in the configuration file record are described in the following table:
+
+| Parameter | Description |
+| :-------- | :---------- |
+| `jwt_map` | The user-name map used to translate claim values to database roles. Optional if you aren't using the default Subject claim values. |
+| `jwt_issuers` | The first part of the discovery URL (`login.microsoftonline.com/<tenant_id>/v2.0`) |
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-aad.md:200]:
```suggestion
| `jwt_issuers` | The first part of the discovery URL (`login.microsoftonline.com/<tenant_id>/v2.0`). |
```

File: docs/content/stable/yugabyte-platform/security/authentication/oidc-authentication-aad.md [hunk 3]
```diff
@@ -185,13 +185,16 @@ For information on configuring flags in YugabyteDB Anywhere, refer to [Edit conf
...
 
-- `jwt_map` - the user-name map used to translate claim values to database roles. Optional if you aren't using the default Subject claim values.
-- `jwt_issuers` - the first part of the discovery URL (`login.microsoftonline.com/<tenant_id>/v2.0`)
-- `jwt_audiences` - the audience or target app for the token, which in this case is the client ID of the application you registered.
-- `jwt_matching_claim_key` - the email attribute you set (for example, `preferred_username`). Optional if you aren't using the default Subject claim values.
-- `jwt_jwks_path` - The JSON Web Key Set (JWKS) is a set of keys containing the public keys used to verify any JWT. These can be uploaded as entries in a single file. When configuring the flag in YugabyteDB Anywhere, click **Add JSON web key set (JWKS)** to upload the JWKS.
+| Parameter | Description |
+| :-------- | :---------- |
+| `jwt_map` | The user-name map used to translate claim values to database roles. Optional if you aren't using the default Subject claim values. |
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/security/authentication/oidc-authentication-aad.md:192]: @Vars-07 do we provide an example of this map file anywhere? What does it look like

**Pull request merged**